client/cody/vscode-codegen/webviews/Chat.tsx

```diff
@@ -63,6 +64,14 @@ export const Chat: React.FunctionComponent<React.PropsWithChildren<ChatboxProps>
 
     const bubbleClassName = (speaker: string): string => (speaker === 'you' ? 'human' : 'bot')
 
+    const scrollToBottom = () => {
+        chatboxRef.current?.scrollIntoView({ behavior: 'smooth' })
```

Comment:
You could `?.` the call so we don't throw when `current` isn't set, if that can happen.

```suggestion
        chatboxRef.current?.scrollIntoView?.({ behavior: 'smooth' })
```